pgcli/pgcompleter.py

```diff
@@ -162,7 +190,7 @@ def reset_completions(self):
                            'datatypes': {}}
         self.all_completions = set(self.keywords + self.functions)
 
-    def find_matches(self, text, collection, start_only=False, fuzzy=True,
+    def find_matches(self, text, collection, mode='name',
```

Comment:
Wouldn't it be better to have `mode` take a string called `'fuzzy'` instead of `'name'`? When `mode` is set to `'name'` is it not self-documenting, it adds a bit of cognitive load.